commit 78cf827bcebfb73ff1a3a6369f9c66693ec33457 (refs/changes/05/2005/2)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2017-05-26T00:02:43+00:00 (2 years, 4 months ago)
    
    joyent/node-sshpk#29 tests fail with openssl 1.1.0e because of extra spaces in DN output
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/test/openssl-cmd.js b/test/openssl-cmd.js
index ab27c93..a62d24f 100644
--- a/test/openssl-cmd.js
+++ b/test/openssl-cmd.js
@@ -222,10 +222,12 @@ function genTests() {
 			var output = Buffer.concat(bufs).toString();
 
 			var m = output.match(/Issuer: (.*)$/m);
-			t.strictEqual(m[1], 'CN=' + algo + ', C=US');
+			t.strictEqual(m[1].replace(/ = /g, '='),
+			    'CN=' + algo + ', C=US');
 
 			m = output.match(/Subject: (.*)$/m);
-			t.strictEqual(m[1], 'CN=' + algo + ', C=US');
+			t.strictEqual(m[1].replace(/ = /g, '='),
+			    'CN=' + algo + ', C=US');
 
 			var re = /DNS:([^, \n]+)([, ]+|$)/gm;
 			m = re.exec(output);
